frontend/src/screens/peers/Peers.tsx

```diff
@@ -87,7 +87,7 @@ export default function Peers() {
         description="Manage your connections with other lightning nodes"
         contentRight={
           <>
-            <Link to="/peers/new">
+            <Link to="/channels/peers/new">
```

Comment:
I don't think this makes sense. I just think we need to be careful if we decide to change the old routes for example (/channels to /node)